packages/cli/src/eventbus/MessageEventBus/MessageEventBus.ts

```diff
@@ -225,7 +223,7 @@ export class MessageEventBus extends EventEmitter {
 
 	private async emitMessage(msg: EventMessageTypes) {
 		if (config.getEnv('endpoints.metrics.enable')) {
```

Comment:
do we still need this check? emitting events when there are no listeners isn't really that expensive 🤔.